### PR TITLE
Pn/feature progress bar

### PR DESCRIPTION
**Pull request by nemoipaha**

### CODE REVIEW

File: resources/assets/templates/mainLayout.hbs [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
         </div>
     </div>
-    <div class="spinner" id="spinner"></div>
+    <!--<div class="spinner" id="spinner"></div>-->
```
Comment (binaryvo) [inline resources/assets/templates/mainLayout.hbs]: Please remove block if it not necessary anymore

File: resources/assets/js/app/views/mainLayout.js [hunk 2]
```diff
@@ -13,24 +13,11 @@ var mainLayoutView = Marionette.LayoutView.extend({
         breadCrumbs: '#breadcrumbs',
         modalWindow: '#modal-container'
     },
+
     ui: {
         spinner: '#spinner'
     },
```
Comment (binaryvo) [inline resources/assets/js/app/views/mainLayout.js]: Probably spinner property not necessary as well

**Comment by binaryvo** [comment]
Please fix remarks and re-open PR

**Comment by nemoipaha** [comment]
Fixed

**Pull request merged**